### PR TITLE
remove dynamic atlas min size limitation

### PR DESCRIPTION
**Pull request by 2youyou2**
Re: cocos-creator/2d-tasks#2175

Changes:
 * remove dynamic atlas min size limitation

### CODE REVIEW

File: cocos2d/core/renderer/utils/dynamic-atlas/atlas.js [hunk 1]
```diff
@@ -54,6 +54,13 @@ cc.js.mixin(Atlas.prototype, {
 
             // texture bleeding
             if (cc.dynamicAtlasManager.textureBleeding) {
+                if (width <= 8 || height <= 8) {
```
Comment (jareguo) [inline cocos2d/core/renderer/utils/dynamic-atlas/atlas.js:58]: 保留注释吧：// Smaller frame is more likely to be affected by linear filter

File: cocos2d/core/renderer/utils/dynamic-atlas/manager.js [hunk 2]
```diff
@@ -116,13 +114,8 @@ let dynamicAtlasManager = {
      * !#zh 可以添加进图集的图片的最小尺寸。
      * @property minFrameSize
      * @type {Number}
+     * @deprecated
      */
-    get minFrameSize () {
-        return _minFrameSize;
-    },
-    set minFrameSize (value) {
-        _minFrameSize = value;
```
Comment (jareguo) [inline cocos2d/core/renderer/utils/dynamic-atlas/manager.js:124]: 加个警告吧？

Comment (2youyou2) [reply cocos2d/core/renderer/utils/dynamic-atlas/manager.js:124]: 加到 deprecated.js 里了

File: cocos2d/core/renderer/utils/dynamic-atlas/atlas.js (hunk 1 above)
Comment (2youyou2) [reply cocos2d/core/renderer/utils/dynamic-atlas/atlas.js:58]: 加了

**Review by SantyWang — APPROVED** [review]

**Review by caryliu1999 — APPROVED** [review]

**Review by jareguo — APPROVED** [review]

**Pull request merged**